backend/controllers/controller.js

```diff
@@ -95,37 +96,57 @@ const factory = ({
       const mb200 = 200000000
 
       if (size > mb200) {
+        logger.error(`A file exceeding ${mb200} was rejected, filename ${offenders.originalFilename} size ${size}`)
         res.status(400)
-        res.send('The csv is to lage. Maximum file size is 200MB')
+        res.send('The csv is too large. Maximum file size is 200MB')
         return
       }
 
       const bytes = fs.readFileSync(offenders.path)
       if (isBinaryFileSync(bytes, size)) {
```

Comment:
Should these 'sync' operations be asynchronous? 
The node docs for 'fs' say: In busy processes, the programmer is strongly encouraged to use the asynchronous versions of these calls. The synchronous versions will block the entire process until they complete — halting all connections.

Reply:
Agreed.